src/store/store.ts

```diff
@@ -9,23 +9,24 @@ const initialState = {
     {id: 2, name: 'Implement Trello Board', description: 'A Kanban board with drag-and-drop feature', status: 'New', assignee: 'Assignee Name'},
     {id: 3 , name: 'Submit code for review', description: 'Open a new pull request', status: 'New', assignee: 'Assignee Name'},
   ],
-  currentDraggedElement: 0
 }
 
 const reducer = (state = initialState, action: AnyAction) => {
   switch(action.type) {
-    case DRAG_BEGIN:
-      return {
-        ...state,
-        currentDraggedElement: action.payload
-      }
     case UPDATE_STATUS:
-      const index = state.currentDraggedElement;
-      const newState = { ...state, currentDraggedElement: null };
-      const newStatus = action.payload;
-      newState.issues[index].status = newStatus;
+      const index = action.payload.issueId;
+      const newStatus = action.payload.newStatus;
+      const newState = state.issues.map(issue => {
```

Comment:
I think **find** is a more suitable method here

Reply:
done